mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -2750,6 +2769,29 @@ func resourceContainerClusterUpdate(d *schema.ResourceData, meta interface{}) er
 		log.Printf("[INFO] GKE cluster %s's cluster-wide autoscaling has been updated", d.Id())
 	}
 
+	if d.HasChange("allow_net_admin") {
```

Comment:
Is this field updatable? If so can you add an update test?

Reply:
Yes the field is updatable. I added update steps to the test to toggle the feature off and on again.